pkg/sensors/tracing/kprobe_test.go

```diff
@@ -6580,3 +6581,249 @@ spec:
 	err = jsonchecker.JsonTestCheck(t, checker)
 	assert.NoError(t, err)
 }
+
+func getArmKprobeSymb(kSymbols *ksyms.Ksyms) string {
+	if kSymbols.IsAvailable("arm_kprobe") == true {
```

Comment:
a nit but you don't need an extra comparison here, as with all the `if` and `else if` with `bool == true`.
```suggestion
	if kSymbols.IsAvailable("arm_kprobe") {
```